### DISCUSSION TITLE
1.12.2

### DISCUSSION THREAD

**Discussion by Lightninge** [Jissee/JarsAuth#14] [Q&A]
Is it possible to make this mod for 1.12.2? I've been searching for this kind mod for ages and would it be so much appreciated

**Comment by Jissee** [comment]
I regret to inform you that I cannot port it back to 1.12.2 (or even 1.16.5). I have tried to do so, but I cannot even establish the development environment (I am using a Mac, and 1.12.2 is not compatible with Apple Silicon). Additionally, the APIs of Minecraft have been changing all the time and it can take bottomless efforts to make the program compatible with those legacy versions. It is time to switch to a more recent version, which has better optimization, performance, game content, and more mods to play with. 

By the way, you are encouraged to do it by yourself and you can ask me if you have issues with programming.
